framework/src/Volo.Abp.AspNetCore/Volo/Abp/AspNetCore/Security/AbpSecurityHeadersOptions.cs

```diff
@@ -1,8 +1,17 @@
+using System.Collections.Generic;
+
 namespace Volo.Abp.AspNetCore.Security;
 
 public class AbpSecurityHeadersOptions
 {
     public bool UseContentSecurityPolicyHeader { get; set; }
 
     public string ContentSecurityPolicyValue { get; set; }
+
+    public Dictionary<string, string> Headers { get; }
```

Comment:
this is not being used

Reply:
Actually it's being used, see: 

https://github.com/abpframework/abp/blob/104498172c9354b9b208fd1a2c909396e2049ed9/framework/src/Volo.Abp.AspNetCore/Volo/Abp/AspNetCore/Security/AbpSecurityHeadersMiddleware.cs#L39